front/Dockerfile

```diff
@@ -1,14 +0,0 @@
-FROM node:12-alpine
-
-WORKDIR /front
-
-COPY citizenlab.config.json citizenlab.config.*.json /
-
-ADD front/package.json front/package-lock.json ./
-ADD front/internals internals
-
-RUN npm install
-
-ADD front/. .
-
-CMD ["npm", "start"]
```

Comment:
@SaraBCL I missed this in my last review, but in fact, we really can't delete this at the moment as it is needed for the open-source images.

In our deployment guide, we're distributing the front-end as a docker image, and to build it CI is using this.

Reply:
Ah, right, well hope it works better in CI than it does in dev ! Any ideas how to work around ? (maybe put this Dockerfile in citizenlab_ee, keep the old one for citizenlab, and replace when enabling ee ? Or is it not worth the extra complexity, and then I close these PRs and just clarify the docs ?

Reply:
I would just do exactly what you did, but for now leave `front/Dockerfile` in place. It's referenced directly [here](https://github.com/CitizenLabDotCo/citizenlab/blob/e2ace51fda90cb1b50a3b53ca1f2a9e3c87e0aa4/.circleci/config.yml#L275), and that should still work.

Reply:
Ah, right, thanks, done !